hm-redirects.php

```diff
@@ -5,7 +5,7 @@
  * @package hm-redirects
  *
  * Description: Handles redirects in a scalable manner.
- * Version: 0.7.0
+ * Version: 0.7.2
```

Comment:
Quick note for future reference - it's best to keep version bumps out of PRs intended to fix a specific problem or add a specific version. That way we don't end up accidentally decreasing the version if we wanted to revert this in a future update.

Reply:
Shall I undo this version bump in this PR, close it out, then create a new PR with an update version and change log. I notice it hasn't been kept up to date with whats in the Tags - https://github.com/humanmade/hm-redirects/tags